src/setuptools_scm/__init__.py

```diff
@@ -188,3 +191,22 @@ def _get_version(config):
         )
 
         return version_string
+
+
+# Public API
+__all__ = [
```

Comment:
I included `NonNormalizedVersion` at the package level but realized (from a flake8 warning stating symbol not used) that there was no `__all__`. I tried to propose one to "officialize" the public symbols.

Reply:
also id like to go for privatizing more of the api